src/main/java/com/github/randoapp/adapter/RandoListAdapter.java

```diff
@@ -95,7 +95,7 @@ public void onBindViewHolder(RandoViewHolder holder, Cursor cursor) {
 
         holder.rando = RandoDAO.cursorToRando(cursor);
         holder.position = cursor.getPosition();
-        holder.timestamp.setText(dateFormat.format(holder.rando.date));
+        holder.timestamp.setText(dateFormat.format(holder.rando.date).toUpperCase());
```

Comment:
can we have 2 formatters: for current year and for past ones? we don't need year for current